src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources_de.properties

```diff
@@ -48,7 +49,6 @@ error.msi-product-version-major-out-of-range=Hauptversion muss im Bereich [0, 25
 error.msi-product-version-build-out-of-range=Build-Teil der Version muss im Bereich [0, 65535] liegen
 error.msi-product-version-minor-out-of-range=Nebenversion muss im Bereich [0, 255] liegen
 error.version-swap=Versionsinformationen f\u00FCr {0} konnten nicht aktualisiert werden
-error.icon-swap=Failed to update icon for {0}
```

Comment:
Why this was removed?

Reply:
This line was added back to all locales as it was a regression from the translation process.